jobs/component_jobs.groovy

```diff
@@ -120,7 +127,7 @@ repos.each { Map repo ->
                   'UPSTREAM_BUILD_URL': '${BUILD_URL}',
                   'UPSTREAM_SLACK_CHANNEL': "${repo.slackChannel}",
                   'COMPONENT_REPO': "${repo.name}",
-                  'ACTUAL_COMMIT': '${ghprbActualCommit}',
+                  'ACTUAL_COMMIT': '${ACTUAL_COMMIT}',
```

Comment:
this is not quite right yet... the export in shell step above doesn't seem to persist when we invoke here